generators/gulp/templates/gulpfile.js

```diff
@@ -66,7 +58,7 @@ gulp.task('test', ['pre-test'], function (cb) {
 
 gulp.task('coveralls', ['test'], function () {
   if (!process.env.CI) {
-    return;
+    return false;
```

Comment:
Is this also fixing to fix an eslint warning? If so, let's edit the config because this is perfectly valid.